src/index.ts

```diff
@@ -22,6 +22,7 @@ import * as downloadUtil from './download-util';
 import * as installUtil from './install-util';
 import { getLatestGcloudSDKVersion } from './version-util';
 import { ExecOptions } from '@actions/exec/lib/interfaces';
+import * as fs from 'fs';
```

Comment:
It looks like we're only using one `fs` method and it's from promises. Can we do `import { promises as fs } from 'fs'` instead?

Reply:
done